Unix/configure

```diff
@@ -154,6 +155,9 @@ do
       certsdir=$arg 
       ;;
 
+    --credsdir=*)
+      credsdir=$arg 
+      ;;
```

Comment:
I removed the lines. As you mention, its validity is limited to --dev, albeit I would not want it simply to be blank.